### ISSUE TITLE
Build with JavaScript mpv plugin support

### ISSUE THREAD

**Issue by Bananaman** [celluloid-player/celluloid#631]
Described in the flatpak issue:

https://github.com/flathub/io.github.celluloid_player.Celluloid/issues/15

Celluloid should be built with JavaScript support to have support for all mpv plugins. Anything that this project can do to improve that, such as by adding instructions for building celluloid with mujs included?

**Comment by gnome-mpv** [comment]
The code that handles plugins in Celluloid is completely language-agnostic; any language supported by `libmpv` will automatically work in Celluloid. To get JavaScript plugins to work, you only need to build `mpv` with JavaScript support enabled.

**Comment by Bananaman** [comment]
@gnome-mpv Ah, thanks for the answer. I suspected that since I saw that Celluloid uses my distro's libmpv which lacks Javascript.

I noticed that Arch builds their libmpv with Javascript. So Ubuntu's repo maintainer probably doesn't know about the problem. The mujs support is pretty hidden in mpv, and only gets added if mujs.h can be found during compile.

Anyway, thanks for the answer. I will close this and refer to the flatpak issue where I hope the internal mpv can be improved to include javascript.
